src/commands/utils/station.ts

```diff
@@ -0,0 +1,44 @@
+import request from 'request';
+import { CommandDefinition } from '../../lib/command';
+import { CommandCategory } from '../../constants';
+import { makeEmbed, makeLines } from '../../lib/embed';
+
+export const station: CommandDefinition = {
+    name: 'station',
+    description: 'Provides station information',
+    category: CommandCategory.UTILS,
+    executor: (msg) => {
+        const splitUp = msg.content.replace(/\.station\s+/, ' ').split(' ');
+
+        if (splitUp.length <= 1) {
+            msg.reply('please provide an ICAO airport code.');
+            return Promise.resolve();
+        }
+        const icaoArg = splitUp[1];
+        request({
+            method: 'GET',
+            url: `https://avwx.rest/api/station/${icaoArg}`,
+            headers: {
+                Authorization: process.env.STATION_TOKEN },
+        }, (error, response, body) => {
+            const stationReport = JSON.parse(body);
+            msg.channel.send(makeEmbed({
+                title: `Station info | ${stationReport.icao}`,
+                description: makeLines([
+                    '**Station Information:**',
+                    `**Name:** ${stationReport.name}`,
+                    `**Country:** ${stationReport.country}`,
+                    `**City:** ${stationReport.city}`,
+                    `**Latitude:** ${stationReport.latitude}`,
+                    `**longitude:** ${stationReport.longitude}`,
+                    ,
```

Comment:
What about adding 
`**elevation_ft:** ${stationReport.elevation_ft:}`,
`**elevation_m:** ${stationReport.elevation_m:}`,
to show the airports' elevation in feet and meters?

Reply:
@GhostEagle68 These will be added in, this command is still in progress while we work out a few things!